File: app/src/main/java/me/pwcong/jpstart/utils/ResourceUtils.java
package me.pwcong.jpstart.utils;

import android.content.Context;

/**
 * Created by Pwcong on 2016/9/24.
 */

public class ResourceUtils {

    private ResourceUtils() {
        throw new RuntimeException("♪(^∇^*)");
    }

    public static String getString(Context context, int resId) {
        return context.getResources().getString(resId);
    }

    public static float getDimension(Context context, int resId) {

        return context.getResources().getDimension(resId);

    }


}

File: app/src/main/java/com/github/pwcong/jpstart/utils/ResourceUtils.kt
package com.github.pwcong.jpstart.utils

import android.content.Context

class ResourceUtils private constructor() {
    init {
        throw RuntimeException("♪(^∇^*)")
    }

    companion object {
        fun getString(context: Context, resId: Int): String {
            return context.resources.getString(resId)
        }

        fun getDimension(context: Context, resId: Int): Float {
            return context.resources.getDimension(resId)
        }
    }
}
